src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6020,14 +6070,14 @@ namespace SimAirServingZones {
 					PreDefTableEntry( pdchS62svrHtEv, FinalSysSizing( AirLoopNum ).AirPriLoopName, EvzMinBySysHeat( AirLoopNum ), 3 ); //Ev
 					PreDefTableEntry( pdchS62svrHtVot, FinalSysSizing( AirLoopNum ).AirPriLoopName, VotHtgBySys( AirLoopNum ), 4 ); //Vot
 					if ( FinalSysSizing( AirLoopNum ).DesHeatVolFlow != 0.0 ) { // Move here from other routine
-						PreDefTableEntry( pdchS62svrHtPercOA, FinalSysSizing( AirLoopNum ).AirPriLoopName, VotHtgBySys( AirLoopNum ) / FinalSysSizing( AirLoopNum ).DesHeatVolFlow ); //%OA
+						PreDefTableEntry( pdchS62svrHtPercOA, FinalSysSizing( AirLoopNum ).AirPriLoopName, 100.0 * VotHtgBySys( AirLoopNum ) / FinalSysSizing( AirLoopNum ).DesHeatVolFlow ); //%OA
```

Comment:
I see this in 62.1-2004 which reports as a fraction. But I also see discussion using "percent outdoor air" in several places. I don't have the latest version of the Standard.

6.2.5.1 Primary Outdoor Air Fraction. When Table 6-3 is used to determine system ventilation efficiency, the zone primary outdoor air fraction (Zp) shall be determined in accordance with Equation 6-5.
Zp = Voz/Vpz (6-5)
where Vpz is the zone primary airflow, i.e., the primary airflow to the zone from the air handler including outdoor air and recirculated return air. Note: For VAV systems, Vpz is the minimum expected primary airflow for design purposes.